### PR TITLE
Actions

### PR DESCRIPTION
**Pull request by jburel**
Replace travis by GH actions
There is still reference to travis e.g. ``travis-fold``, ``travis-build`` since it will require some change in other repositories

cc @sbesson

### CODE REVIEW

**Comment by joshmoore** [comment]
Looks like the solution for travis-fold is "echo ::group...":

https://github.community/t/has-github-action-somthing-like-travis-fold/16841/2

**Comment by jburel** [comment]
I was more thinking of impact now on file like https://github.com/ome/openmicroscopy/blob/develop/components/tools/travis-build
 A modification in this repo, implies synchronising other repo at the same time

File: README.md [hunk 1]
```diff
@@ -198,5 +198,5 @@ Examples
    - https://github.com/ome/scripts
 
 You can find more examples by searching for
-"[openmicroscopy/omero-test-infra filename:.travis.yml](https://github.com/search?q=openmicroscopy%2Fomero-test-infra+filename%3A.travis.yml&type=Code)"
+"[ome/omero-test-infra filename:.omero_plugin.yml](https://github.com/search?q=openmicroscopy%2Fomero-test-infra+filename%3A.omero_plugin.yml&type=Code)"
```
Comment (joshmoore) [inline README.md:201]: Probably get rid of the leading `.`?

File: README.md [hunk 2]
```diff
@@ -156,7 +156,7 @@ Run the linter, install, run tests
 Remember, you may need to set some environment variables to ensure your plugin
 is discovered and tested correctly.
 
-If you are modifying an existing plugin see `.travis.yml` for the required variables.
+If you are modifying an existing plugin see `.omero_plugin.yml` for the required variables.
```
Comment (joshmoore) [inline README.md:159]: Did you mean for the file in this repo to be named "omero_plugin.yml" rather than "main.yml"? (Also, don't need the leading dot)

**Review by joshmoore — COMMENTED** [review]
Few comments. Unfortunately, the build's not running here yet.

File: README.md [hunk 3]
```diff
@@ -198,5 +200,5 @@ Examples
    - https://github.com/ome/scripts
 
 You can find more examples by searching for
-"[openmicroscopy/omero-test-infra filename:.travis.yml](https://github.com/search?q=openmicroscopy%2Fomero-test-infra+filename%3A.travis.yml&type=Code)"
+"[ome/omero-test-infra filename:omero_plugin.yml](https://github.com/search?q=openmicroscopy%2Fomero-test-infra+filename%3Aomero_plugin.yml&type=Code)"
```
Comment (sbesson) [inline README.md:203]: This query returns nothing, did you mean https://github.com/search?q=omero-test-infra+filename%3Aomero_plugin.yml&type=Code ?

File: README.md [hunk 4]
```diff
@@ -56,7 +58,7 @@ It is also possible to combine several application e.g.
 but each of the applications must pass when it is run alone as well.
 
 These commands should be invoked by the `script` step in the project's
-.travis.yml file. Set any environment variables as necessary.
+.omero_plugin.yml file. Set any environment variables as necessary.
```
Comment (sbesson) [inline README.md:61]: Still need to get rid of the leading dot ?

**Review by sbesson — COMMENTED** [review]
https://github.com/jburel/omero-test-infra/runs/1520247194?check_suite_focus=true is passing so the migration to GitHub actions is effective.

Reading the README, I am slightly confused between what is meant to refer to the workflow file in this repository vs the workflow file in a plugin consuming omero-test-infra and/or the [template workflow](https://github.com/ome/.github/blob/master/workflow-templates/omero_plugin.yml).

File: README.md (hunk 4 above)
Comment (jburel) [reply README.md:61]: hummm....

**Comment by jburel** [comment]
@sbesson README updated

**Review by sbesson — APPROVED** [review]

**Pull request merged**